index.js

```diff
@@ -72,7 +75,10 @@ module.exports = (pluginOptions = {}) => (nextConfig = {}) => {
             return extractFrontMatter(pluginOptions, files, compilation.context)
           },
           files: {
-            pattern: '**/*.mdx',
+            pattern:
+              pluginOptions.fileExtensions.length > 1
+                ? `**/*.{${pluginOptions.fileExtensions.join(',')}}`
```

Comment:
Oh huh, does that extended glob syntax misbehave if there's only one thing in it? Or is there another reason for separating them?

Reply:
yeah, it didn't work with only one item - you can [test it here](http://www.globtester.com/#p=eJzT19LS19KrziquBQAMYgLg&r=eJyzScksU0jOSSwutlVSstMPLU4tKtZPyU9P1S9LLarUT8xNrErVyyq20Qeqs7NBVVxaXKSfk5%2BcmKOflJmnX55fDlEFALKDHMQ%3D&) - thank goodness for tests!